.travis.yml

```diff
@@ -0,0 +1,16 @@
+notifications:
+  email: false
+
+sudo: true
+
+language: python
+
+services:
+  - docker
+
+install:
+  - docker build -t worldpopulation .
+  - docker run -v `pwd`:/code -i -t --rm worldpopulation jupyter nbconvert world_population_analysis.ipynb
+
+script:
+  - ls -l world_population_analysis.ipynb
```

Comment:
Good idea; although validate does not return proper exit status. I'll open an issue in the yadage repository.